src/command/init/helpers.rs

```diff
@@ -61,13 +61,10 @@ pub fn generate_project_created_message(
     ));
     output.push('\n');
 
-    // Add warning section
     output.push_str(&format!(
-        "{}\n",
-        Style::WarningHeading.paint("️▲ Before you proceed:")
+        "Writing {} to .env\n\n",
+        Style::GraphRef.paint("APOLLO_KEY"),
```

Comment:
This is printed after we have done this, so we can just say:


```suggestion
        "Your GraphOS credentials can be find in your `.env` file.\n\n"
```

Reply:
or something like that

Reply:
Great idea, updated!